src/modules/pmc/src/lib/ExecutionState.h

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#include <string>

```

Comment:
Try to include headers from source files, not from headers. Move this out from here.